actions/class.Runner.php

```diff
@@ -67,6 +103,11 @@ protected function getServiceContext($check = true)
         if (!$this->serviceContext) {
             $testDefinition = $this->getRequestParameter('testDefinition');
             $testCompilation = $this->getRequestParameter('testCompilation');
+            $csrfToken = $this->getRequestParameter('x-security-token');
+
+            if (!$this->getCsrf()->checkCsrfToken($csrfToken)) {
+                throw new common_exception_InconsistentData('CSRF attempt!');
```

Comment:
What's the effect of this exception ? Is the exception caught ?
The expected behavior would be to
1. log the XSRF attempt (with an error level)
2. response with a 403 status code but without any explicit reason.